Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;

```

Comment:
The value of the 'padding' property should include units (e.g., pixels) such as '10px' for consistency and to ensure proper rendering.